frontend/src/components/TheHeader.vue

```diff
@@ -1,13 +1,18 @@
 <template>
   <header class="p-2 flex justify-between items-center bg-slate-800 text-white">
     <div class="inline-flex flex-row items-center">
-      <a href="https://dbcost.com" target="_blank">
-        <img
-          class="h-8"
-          src="../assets/icon/dbcost-logo-full.png"
-          alt="Bytebase"
-        />
-      </a>
+      <n-button
+        :bordered="false"
+        @click="() => $router.push({ name: 'dashboard' })"
+      >
+        <a href="https://dbcost.com" @click.prevent="">
```

Comment:
I guess we don't need <a> ?

Reply:
The btn span  is for consistency in CSS style and the click behavior of the btn span is for consistency in code  style (otherwise we may need to leave a extra comment to explain).

Reply:
I don't get this the extra <a> seems to be redundant since we already have @click="() => router.push